ogusa/tests/test_parameters.py

```diff
@@ -58,8 +58,8 @@ def test_parameters_metadata_baseline():
     [('', None, "./TxFuncEst_{}.pkl"),
      ('', "test.pkl", "test.pkl"),
      (9, None, "./TxFuncEst_{}9.pkl")])
-def test_tx_func_est_path((monkeypatch, baseline, guid, tx_func_est_path,
-                          exp_tx_func_est_path)):
+def test_tx_func_est_path(monkeypatch, baseline, guid, tx_func_est_path,
```

Comment:
@hdoupe I need some help fixing  this test.  I'm a bit confused by what it is trying to do as it's not testing directly a function in `parameters.py`.  Do I need some pickle files sitting in specific locations?